apps/labs/posts/the-new-spyder-editor-documentation-under-the-spotlights.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'The new Spyder Editor documentation under the spotlights!'
+published: October 31, 2022
+author: hanan-younes
+description: 'In this blogpost, I share my experience as a Google Season of Docs 2022 technical writer working on updating the Editor user documentation.'
+category: [Community, OSS Experience]
+featuredImage:
+  src: /posts/the-new-spyder-editor-documentation-under-the-spotlights/blog_feature_var3.svg
+  alt: 'An illustration of a brown and a white hand coming towards each other to pass a business card with the logo of Quansight Labs'
+hero:
+  imageSrc: /posts/what-do-we-need-to-write/blog_hero_var3.svg
```

Comment:
I think the path here is mis-specified 😉  If you update the folder name the hero appear.